src/toil/wdl/wdltoil.py

```diff
@@ -5500,6 +5501,29 @@ def main() -> None:
                         )
                         options.all_call_outputs = True
 
+                WDL.Lint.lint(document)
```

Comment:
This could use a comment about how this mutates `document` to add linting information, and doesn't itself print any errors or stop the workflow.